android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -724,11 +724,11 @@ public double getTotalMemorySync() {
   @SuppressWarnings({"ConstantConditions", "deprecation"})
   public String getInstanceIdSync() {
     try {
-      if (Class.forName("com.google.android.gms.iid.InstanceID") != null) {
-        return com.google.android.gms.iid.InstanceID.getInstance(getReactApplicationContext()).getId();
+      if (Class.forName("com.google.firebase.iid.FirebaseInstanceId") != null) {
+        return com.google.firebase.iid.FirebaseInstanceId.getInstance().getId();
```

Comment:
I'll have to do some research quickly to make sure these are exactly equivalent - but in your research is that your conclusion? People rely on these ids being exact + precise for device id